controllers/xmlrpc/src/main/java/orca/controllers/xmlrpc/geni/GeniAmV2Handler.java

```diff
@@ -807,7 +807,14 @@ public static GeniStates getSliceGeniState(XmlrpcOrcaState instance, String slic
                     if (r.getState() == OrcaConstants.ReservationStateFailed) {
                         en.put(ApiReturnFields.GENI_ERROR.name, (r.getNotices() != null ? r.getNotices()
                                 : "ERROR: no detailed error message available"));
-                    } else {
+                    } 
+                    else if (r.getState() == OrcaConstants.ReservationStateClosed && r.getNotices() != null &&
+                            r.getNotices().toLowerCase().contains("insufficient")) {
+                        en.put(ApiReturnFields.GENI_ERROR.name, (r.getNotices() != null ? r.getNotices()
+                                : "ERROR: no detailed error message available"));
```

Comment:
I think maybe the error should be: 

String errorDetail = r.getNotices();
if (errorDetail == null) 
  errorDetail = "no further information available";
String errorMessage = "ERROR: unable to provision requested resource (" + errorDetail  + ")";
...
I can't remember how detailed our notices are - just trying to make it as clear as possible to users.

Reply:
okay, Notices contain both the state and any error messages if available. Will make the changes.


Example notice from the issue reported:


```
pequod:show>set current slices
OK
pequod:show>show reservations for current actor ufl-sm 
Reservations for slice d7dd3631-7f84-4446-a41a-cfbab4c08d2c:
51edc1c8-c61f-4365-8249-7dcb25469d92	ufl-sm
	Slice: d7dd3631-7f84-4446-a41a-cfbab4c08d2c
	1	uflvmsite.vlan	[ closed, nascent]	
	Notices: Reservation 51edc1c8-c61f-4365-8249-7dcb25469d92 (Slice urn:publicid:IDN+ch.geni.net:GENI-Enter+slice+aslice2) is in state [Closed,None]
	Start: Fri Jul 17 14:59:34 EDT 2020	End: Fri Jul 24 14:59:11 EDT 2020	Requested end: Fri Jul 24 14:59:11 EDT 2020
6ec71284-d2d2-479f-9f13-6f28518f1353	ufl-sm
	Slice: d7dd3631-7f84-4446-a41a-cfbab4c08d2c
	1	uflvmsite.vm	[ closed, nascent]	
	Notices: Reservation 6ec71284-d2d2-479f-9f13-6f28518f1353 (Slice urn:publicid:IDN+ch.geni.net:GENI-Enter+slice+aslice2) is in state [Closed,None]
Last ticket update: java.lang.RuntimeException: Insufficient <memoryCapacity,1000> to meet request:3000
	Start: Fri Jul 17 14:59:35 EDT 2020	End: Fri Jul 24 14:59:11 EDT 2020	Requested end: Fri Jul 24 14:59:11 EDT 2020
```